openapi-generator/src/main/java/io/micronaut/openapi/generator/AbstractMicronautJavaCodegen.java

```diff
@@ -1071,7 +1071,7 @@ private String makeSureImported(String typeName, Set<String> imports) {
     public String toVarName(String name) {
         var varName = super.toVarName(name);
 
-        if (varName.chars().allMatch(Character::isUpperCase)) {
+        if (varName.chars().allMatch(c -> Character.isUpperCase(c) || c == '_')) {
```

Comment:
why are you adding these changes to a PR about updating versions? Mixing changes makes reviews really hard.

Reply:
because this fix for openapi-generator 7.5.0, without this fix you can't use 7.5.0